main/src/ca/uwaterloo/flix/language/errors/LexerError.scala

```diff
@@ -259,17 +322,16 @@ object LexerError {
   }
 
   /**
-    * An error raised when an unterminated string is encountered.
-    *
-    * @param loc The location of the opening `{`.
-    */
+   * An error raised when an unterminated string is encountered.
+   *
+   * @param loc The location of the opening `{`.
+   */
   case class UnterminatedStringInterpolation(loc: SourceLocation) extends LexerError with Recoverable {
     override def summary: String = s"Unterminated string interpolation."
 
     override def message(formatter: Formatter): String = {
       import formatter._
       s"""${line(kind, source.name)}
-         |>> Missing '}' in string interpolation.
```

Comment:
@herluf-ba Was this intended?

Reply:
No that seems weird, I must have yanked that line by accident.